test/unit/test_arguments.py

```diff
@@ -87,6 +87,7 @@ def test_device_args(test_params, expected_params):
               transformer_no_positional_encodings=False,
               attention_use_prev_word=False,
               rnn_decoder_zero_init=False,
+              rnn_decoder_state_init="last",
```

Comment:
C.RNN_DEC_INIT_LAST

Reply:
not using the defined constants is intentional here (also done for other args in the same test) so that we get a failure if we change the constant definition.

Reply:
Right, we are in the test. Silly me :-)